src/mam4xx/mo_gas_phase_chemdr.hpp

```diff
@@ -148,7 +149,9 @@ void perform_atmospheric_chemistry_and_microphysics(
     const View1D &work_set_het, const seq_drydep::Data &drydep_data,
     const View1D &aqso4_flx, const View1D &aqh2so4_flx,
     Real dvel[gas_pcnst], // deposition velocity [cm/s]
-    Real dflx[gas_pcnst], mam4::Prognostics &progs) {
+    Real dflx[gas_pcnst],
+    // out
+    View3D &qgcm_tendaa, View3D &qqcwgcm_tendaa, mam4::Prognostics &progs) {
```

Comment:
I think passing these Views this way is the right way to do it in spirit. I don't know what the "blessed" way of doing it these days is, because
1. I believe you can omit the `&`, as Views wrap pointers
2. IIRC, you can also stick a `const` on them even when you change their contents, because their `const`ness is not determined by that of their underlying storage.

But I don't see anything wrong with this.